app/src/main/java/com/example/reflector_android/network/RSSService.kt

```diff
@@ -15,9 +16,11 @@ class RSSService {
             if (!response.isSuccessful) {
                 throw IOException("Unexpected code $response")
             }
+
             val body = response.body!!.byteStream()
```

Comment:
This is considered to be "Unsafe".
There can be a case where response.body == nil, which will cause this line to crash when using !! to unwrap it. Check to see if body is nil before force unwrapping.
```
if response.body == nil {
return nil
}
 val body = response.body!!.byteStream()
```